script.js

```diff
@@ -19,21 +44,32 @@ function buildGrid(size) {
 
 buildGrid(16);
 
+//
+/**
+ * Nice! You can do this in a one-liner if you want to be fancy and use more modern JS
+ *
+ * Becomes this: const getRandomColor = () => `#{Math.floor(Math.random() * 16777215).toString(16)}`;
+ *
+ * But that's kind of hard to read when you're first learning the language
+ */
```

Comment:
I do understand that example syntax. I guess I don't think to store the function in a const variable because for some reason I think it to be immutable versus let?

Reply:
A.) Oops my syntax is wrong - JS isn't liking the template strings: `const getRandomColor = () => '#' + Math.floor(Math.random() * 16777215).toString(16)};`

Ah I think the instincts here, and it's a good thought... you could easily swap out `let` for `const` here if you wanted to. But more importantly, _is_ it immutable?

This gets more into TypeScript land than anything else and into the weeds of types and immutability, but think about the value of this:
```js
const getRandomColor = () => '#' + Math.floor(Math.random() * 16777215).toString(16)};
```
`getRandomColor`'s value never changes - it's always a what? 

...It's a function. And that function returns a string. Now because of the `Math.random`, we can be reasonably assured that the function's return type will always yield a new, random value. But the variable itself - `getRandomColor`, is always the same.

Try this out in your console:

```js
const getRandomColor = () => '#' + Math.floor(Math.random() * 16777215).toString(16);

// now log it out once or twice - just the variable, not the invocation of the function
console.log(getRandomColor);
console.log(getRandomColor);

// now log out the invocation a couple of times
console.log(getRandomColor());
console.log(getRandomColor());
console.log(getRandomColor());
```